README.md

```diff
@@ -342,3 +342,9 @@ A module containing [examples](examples/README.md) of the project usage.
 
 ## How to contribute
 Please see our [**Contribution Guidelines**](CONTRIBUTING.md).
+
+
+# Extras
+ - For Menas migration, there is a useful script available in
+[scripts/migration/migrate_menas.py](scripts/migration/migrate_menas.py)
+   (you need python packages `pymongo` and `minydra` to run) 
```

Comment:
I like this. Could we also commit requirements.txt? 
[popreqs](https://github.com/bndr/pipreqs) will generate them for you, so in the end, users just run `pip install -r requirements.txt`

Reply:
wow, did not know this, I will try it, nice! :D

Reply:
Great idea, done.